src/App.css

```diff
@@ -1,3 +1,64 @@
 .App {
-  text-align: center;
+  width: 100%;
+  height: 100vh;
```

Comment:
Do we really need to define it if we set its child width to 100vh? (mapContainer)

Reply:
You're not clear, width cannot be vh.
Its either in % or it is height.